docusaurus/docs/how-to-guides/data-source-plugins/config-test.md

```diff
@@ -0,0 +1,135 @@
+---
+id: data-source-configuration
+title: Configure data sources
```

Comment:
The current title makes it sound like we are going to teach developer how to configure their data sources instead of how to create good data source configuration options

Reply:
Agreed. I'm not sure that "pages" is the best word, but I don't have a better one at the moment.